docs/backlog.md

```
@@ -519,6 +519,102 @@ superfície agora — mas a tabela que a `900-21b` cria já é o destino final d
+
+### [Epic 900] 🟡 A exceção referencial do trigger de `platform_audit_log` deixa o DONO nulificar atribuição
+
+**Adicionado em:** 2026-08-30
+**Prioridade:** P3 — residual conhecido, com mitigação já no lugar
+**Origem:** Camada B da Story `900-51` (achado do `@dev` durante a implementação)
+
+Os dois consertos que o `@po` pediu — `ON DELETE SET NULL` nas FKs (R7, para a trilha sobreviver à
+org e ao usuário) e trigger de imutabilidade (R1) — **colidiam**, e nenhuma leitura do código
+revelava isso. Medido só ao rodar a Camada B contra o Postgres de verdade:
+
+    DELETE FROM users WHERE id = <ator que já agiu>;
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)